test/Groups/elements.jl

```diff
@@ -32,7 +34,11 @@
   G=symmetric_group(6)
   x=gap_perm([2,3,4,5,6,1])
   @test x==perm(G,[2,3,4,5,6,1])
+  @test x==gap_perm(fmpz.([2,3,4,5,6,1]))
+  @test x==perm(G,fmpz.([2,3,4,5,6,1]))
+  @test cperm(G,Int[])==one(G)
   @test x==cperm(G,1:6)
+  @test x==cperm(G,fmpz(1):fmpz(6))
```

Comment:
I really don't think we need to support fmpz step ranges like this. And if we do, I'd rather add a dedicated iterator and/or collect method for `StepRange{fmpz}` to fix it

Reply:
I had just removed it.